rocky/rocky/views/ooi_list.py

```diff
@@ -142,6 +143,37 @@ def _set_scan_profiles(self, selected_oois: List[Reference], request: HttpReques
         )
         return self.get(request, *args, **kwargs)
 
+    def _set_oois_to_inherit(
+        self, selected_oois: List[Reference], request: HttpRequest, *args, **kwargs
+    ) -> HttpResponse:
+        for ooi in selected_oois:
+            try:
+                self.octopoes_api_connector.save_scan_profile(
+                    EmptyScanProfile(reference=Reference.from_str(ooi)),
+                    valid_time=datetime.now(timezone.utc),
+                )
+            except (RequestException, RemoteException, ConnectionError):
+                messages.add_message(
+                    request,
+                    messages.ERROR,
+                    _("An error occurred while setting clearance level to inherit for %s.") % ooi,
+                )
+                return self.get(request, status=500, *args, **kwargs)
+            except ObjectNotFoundException:
+                messages.add_message(
+                    request,
+                    messages.ERROR,
+                    _("An error occurred while setting clearance level to inherit for %s.") % ooi
+                    + _("OOI doesn't exist"),
```

Comment:
Usually, there should have one string with formatting applied. Or are these really separate messages?